src/vs/base/browser/ui/sash/sash.ts

```diff
@@ -208,93 +208,95 @@ export class Sash extends Disposable {
 			}
 		}
 
-		if (!this.state) {
```

Comment:
Are these changes intentional? They look like they just change the conditional but it makes this pr really difficult to review

Reply:
I sent the changes from the wrong branch, so I fixed that now.